googlechat/main_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestWriteMessage(t *testing.T) {
+	/*
```

Comment:
hmm, why are these commented out?

Reply:
These were originally from the Slack version, just bitrot. I'll work at adding unit tests